### DISCUSSION TITLE
Request for Pre-trained DPA-1 Model Weights on OCP Dataset

### DISCUSSION THREAD

**Discussion by Andy6M** [deepmodeling/deepmd-kit#2795] [Q&A] [answered]
Dear author:

Hello,

I hope this finds you well. I am currently working on a project where I am interested in utilizing the DPA-1 model that's pre-trained on the OCP dataset. My aim is to apply it to downstream tasks involving molecules or MOF systems. Unfortunately, I couldn't find the trained model weights in the repository.

Would it be possible to provide the weights or guide me to where I might find them? If we end up using these weights for our project, we will ensure appropriate citation, benefiting both our research and the contributors of this repository.

Thank you for your time and consideration. I look forward to your response.

Best regards,
AndyXie

**Comment by owen-rett** [comment, marked as answer]
See this issue: 
https://github.com/deepmodeling/deepmd-kit/discussions/2249#discussioncomment-4674060
